lib/crypto-algorithms/olm.js

```diff
@@ -185,4 +185,5 @@ OlmDecryption.prototype.decryptEvent = function(event) {
     }
 };
 
-base.registerAlgorithm(olmlib.OLM_ALGORITHM, OlmEncryption, OlmDecryption);
+base.registerAlgorithm(olmlib.OLM_V2_ALGORITHM, OlmEncryption, OlmDecryption);
```

Comment:
Can we just force a logout/login cycle on the clients to give them a new device-ID instead? I'm really not happy about bumping an unrelated crypto algorithm version number here.